core/src/main/java/org/elasticsearch/search/aggregations/metrics/sum/Sum.java

```diff
@@ -29,4 +33,13 @@
      * The sum.
      */
     double getValue();
+
+    // TODO we could pull common xContent rendering into a static method in the interface
```

Comment:
This was an example of how we could share the xContent rendering logic between InternalXY and ParsedXY aggregations. I don't know if it would work in all places. I only did it for SUM, I could try to do it for Min/Max/Avg etc but that means slight changes in the existing internal aggregations. Not sure which way to go here atm, maybe you have some opinion?